src/button/ux-button.ts

```diff
@@ -29,12 +29,24 @@ export class UxButton implements Themable {
     if (this.theme) {
       this.styleEngine.applyTheme(this, this.theme);
     }
+
+    if (this.disabled || this.disabled === '') {
+      this.button.setAttribute('disabled', '');
+    }
   }
 
   public themeChanged(newValue: any) {
     this.styleEngine.applyTheme(this, newValue);
   }
 
+  public disabledChanged(newValue: any) {

```

Comment:
Should we also adjust this casting to match what we've done for the disabled attribute?  So this would now become:
`public disabledChanged(newValue: boolean | string)`?